app/models/null_bid.rb

```diff
@@ -0,0 +1,7 @@
+class NullBid
+  def amount
```

Comment:
I'd be 👍 for explicit `nil` here. Otherwise it might make a future dev think we just forgot to fill in these methods. Explicit `nil` says, no, this is what the method should be.